### PR TITLE
Sup 670 kb add where are my backups stored help article

### PR DESCRIPTION
**Pull request by cybermax-008**
Users are interested to know where are their backups stored in. This is causing unnecessary contacts to support. By adding the information on where to see the location of backup will be easier for customers to learn more about it.

### CODE REVIEW

File: docs/platform/concepts/service_backups.rst [hunk 1]
```diff
@@ -167,6 +167,14 @@ Aiven for ClickHouse backups contain database lists, table schemas, table conten
 
     For more information on Aiven for ClickHouse backups, see :ref:`Backup and restore <backup-and-restore>`.
 
+Where do we store the backups?
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service_backups.rst:170]: Please remove title "Where do we store the backups?".

File: docs/platform/concepts/service_backups.rst [hunk 2]
```diff
@@ -167,6 +167,14 @@ Aiven for ClickHouse backups contain database lists, table schemas, table conten
...
+------------------------------
+
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service_backups.rst:172]: Please move the whole paragraph and the screenshot to section "About backups at Aiven". Include them after sentence "All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored."

File: docs/platform/concepts/service_backups.rst (hunk 1 above)
Comment (harshini-rangaswamy) [inline docs/platform/concepts/service_backups.rst:170]: Suggest rewording the subheading to "Backup storage location" to align with the article's headings and tone.

File: docs/platform/concepts/service_backups.rst [hunk 3]
```diff
@@ -167,6 +167,14 @@ Aiven for ClickHouse backups contain database lists, table schemas, table conten
...
+The backups are stored encrypted in the object storage of the cloud region where the service is running at (Example: S3 for AWS, GCS for GCP). The location of the backups can be seen under **Backups** tab in Aiven Console as shown below,
```
Comment (harshini-rangaswamy) [inline docs/platform/concepts/service_backups.rst:173]:
```suggestion
The backups are securely encrypted and stored in the object storage of the cloud region where the service is running. For example, AWS uses S3 (Simple Storage Service), while GCP (Google Cloud Platform) uses GCS (Google Cloud Storage). To view the location of the backups, you can navigate to the **Backups** tab in the Aiven Console.

Comment (wojcik-dorota) [inline docs/platform/concepts/service_backups.rst:173]:
```
Backups at Aiven are stored in the object storage of the cloud region where a service runs (for example, S3 for AWS or GCS for GCP). You can check the location of your service's backups in `Aiven Console <https://console.aiven.io/>`_ > your service's homepage > **Backups** tab.
```

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
HI @cybermax-008, thank you for making these changes. Added some suggestions to align the content's voice and tone.

**Comment by wojcik-dorota** [comment]
@cybermax-008 thank you for adding this info! I suggest moving this content to a different section. There are also a few editorial changes I requested.

File: docs/platform/concepts/service_backups.rst [hunk 4]
```diff
@@ -6,9 +6,12 @@ This article provides information on general rules for handling service backups
...
+All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. Backups at Aiven are stored in the object storage of the cloud region where a service runs (for example, S3 for AWS or GCS for GCP). You can check the location of your service's backups in `Aiven Console <https://console.aiven.io/>`_ > your service's homepage > **Backups** tab.
 
-Aiven takes service backups for managing purposes. These backups are compressed and encrypted by the Aiven management platform and, as such, are not available for download for any service type.
+.. image:: /images/platform/concepts/backup_location_preview.png
+    :alt: Backup location preview on console
+
+The backup retention times vary based on the service and the selected service plan. Aiven takes service backups for managing purposes. These backups are compressed and encrypted by the Aiven management platform and, as such, are not available for download for any service type.
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service_backups.rst:14]: Apologies! One more - please put sentence "The backup retention times vary based on the service and the selected service plan." on a separate line.

The next sentence ("Aiven takes service backups...") dwells on something different and should be in a separate paragraph below.

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

File: docs/platform/concepts/service_backups.rst (hunk 4 above)
Comment (cybermax-008) [reply docs/platform/concepts/service_backups.rst]: Thanks for letting me know! I have made the changes now.

**Review by wojcik-dorota — APPROVED** [review]
